plugins/modules/dcnm_policy.py

```diff
@@ -715,7 +762,11 @@ def dcnm_policy_get_diff_merge(self):
 
             rc, policy_id = self.dcnm_policy_compare_policies(policy)
 
-            if rc == "DCNM_POLICY_ADD_NEW":
+            if rc == "DCNM_POLICY_DUPLICATED":
+                self.module.fail_json(
+                    f"Policies with the same description are found in DCNM/NDFC: {self.use_desc_as_key}, {policy['description']}"
```

Comment:
Suggest adding this at the beginning 

`f"Multiple policies found with the same description in DCNM/NDFC...`

Just makes it a bit more clear, since one is ok for modification

Reply:
Done